src/electron/platform/android/scan-results.ts

```diff
@@ -22,4 +25,13 @@ export class ScanResults {
             return null;
         }
     }
+
+    public get ruleResults(): RuleResultsData[] {
+        try {
+            const results = this.rawData.axeRuleResults;
+            return results ? results : [];
```

Comment:
FYI (Not necessary, this pattern is fine), `return results || []` is another common pattern - the `||` operator will return the first 'truthy' value it finds and `null` is not truthy

Reply:
Thanks! Will include this in the next iteration if there is one or in next PR if not. Next PR will need to touch this file anyway.

Reply:
Fixed